specification.md

```diff
@@ -4088,8 +4100,7 @@ If it is set to `continue`, if the parent workflow completes, the subflow execut
 
 | Parameter | Description | Type | Required |
 | --- | --- | --- | --- |
-| errorRef | Reference to a unique workflow error definition | string | yes if errorRefs is not used |
-| errorRefs | Reference one or more unique workflow error definition | array | yes if errorRef is not used |
+| errorRef or errorRefs | Reference one unique workflow error definition, or multiple unique workflow error definitions | string (errorRef) or array (errorRefs) | yes |
```

Comment:
Are ```errorRef``` and ```errorRefs``` both allowed to be specified in a single Error Definition?

Reply:
If I understood your question correctly, no. And below we have the explanation:

```
Note that the `errorRef` and `errorRefs` properties are mutually exclusive, meaning that you can only specify one or the other, but not both at the same time.
```

Reply:
@DoisKoh 

> Also, is the actually "ErrorRef Definition" rather than "Error Definition"?

not sure i understand, these props are part of the "Error" definition - https://github.com/serverlessworkflow/specification/blob/main/schema/workflow.json#L327

maybe I am missing something

Reply:
Ah right, well it's a bit confusing because... there's separate "ErrorDef" definition, thought it might be more apt to have different names

https://github.com/serverlessworkflow/specification/blob/main/schema/errors.json